src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -56,6 +57,91 @@
                                        ["code", "lib", "func", "reason"])
 
 
+def _make_asn1_int(backend, x):
+    i = backend._lib.ASN1_INTEGER_new()
+    # i = backend._ffi.gc(i, backend._lib.ASN1_INTEGER_free)
```

Comment:
If this isn't being gc'd please put a comment stating that at the top of the function and remove this line.